actioncable/lib/action_cable/connection/base.rb

```diff
@@ -126,7 +126,8 @@ def on_message(message) # :nodoc:
       end
 
       def on_error(message) # :nodoc:
-        # ignore
+        # log errors to make diagnosing socket errors easier
+        logger.error "Error writing to Action Cable socket: #{message}"
```

Comment:
Did a quick read through of websocket-driver, and it looks like `on_error` could also be called when protocol errors occur, etc. Can we try and make the message a little more general?